sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -261,6 +276,63 @@ def fetchall(
         )
         return list(self._query_data)
 
+    def __add_nested_columns(
```

Comment:
Why double underscore?

Reply:
My bad meant to have a single underscore